config/config_test.go

```diff
@@ -51,3 +51,7 @@ func TestNewSDK(t *testing.T) {
 		require.Equal(t, tt.wantShutdownErr, sdk.Shutdown(context.Background()))
 	}
 }
+
+func toPtr[V int | string](in V) *V {
```

Comment:
Can we rename it to `ptr`? Simiarly in Go we have `String` and not e.g. `ToString`.

Is there any reason why we constrain the type parameters to only `int` or `string`?

```suggestion
func ptr[T any](v T) *T {
	return &v
}
```
```